### ISSUE TITLE
VSCode language server panic in resolving.rs

### ISSUE THREAD

**Issue by mitchgrout** [slint-ui/slint#5652]
VSCode extension for slint crashed while editing a .slint file. Extension version was v1.7.0, and the error output:

```
thread 'main' panicked at internal\compiler\passes\resolving.rs:169:13:
assertion failed: ctx.diag.has_errors()
```

This was triggered by creating a preview for the following component:

```js
import { Button } from "std-widgets.slint";

export component ExportedComponent {
  width: 80px;
  height: 60px;
  Button { }
}
```

...then in the preview, enabling the editor, selecting the button, and setting the ~`value`~ `accessible-action-set-value` property to `:`. I haven't checked all possibilities, but it seems that anything that isn't strictly letters will cause a panic.

![image](https://github.com/user-attachments/assets/2e9cb31b-9fa1-4db1-b676-fed292f50783)

**Comment by ogoffart** [comment]
Thanks for reporting a bug.

I can reproduce it in slintpad as well.

There are two problem:
 - accessibe-action-* are callback and shouldn't be in that list
 - even then, this should cause an error (and not a panic) when building the object_tree, before we reach the resolving phase.
